### PR TITLE
fix reading from ~/.z3.cfg file for config and pep8.

### PR DESCRIPTION
**Pull request by AMecea**
Make z3 to search for `~/.z3.cfg` config file.

### CODE REVIEW

File: z3/pput.py [hunk 1]
```diff
@@ -278,7 +278,7 @@ def parse_args():
         description='Read data from stdin and upload it to s3',
         epilog=('All optional args have a configurable default. '
                 'Order of precedence is command line args then '
-                'environment variables then user config ~/.z3.cfg'
+                'user config ~/.z3.cfg then environment variables then '
```
Comment (rspeed) [inline z3/pput.py]: There's an extra "then " at the end of this line. In the current state, this evaluates to (emphasis mine):

> Order of precedence is command line args then user config ~/.z3.cfg then environment variables **then  then** default config.

The space at the beginning of the next line should also be removed.

**Comment by vtemian** [comment]
LGTM

File: z3/pput.py [hunk 2]
```diff
@@ -278,8 +278,8 @@ def parse_args():
         description='Read data from stdin and upload it to s3',
         epilog=('All optional args have a configurable default. '
                 'Order of precedence is command line args then '
-                'environment variables then user config ~/.z3.cfg'
-                ' then default config.'),
+                'user config ~/.z3.cfg then environment variables '
```
Comment (rciorba) [inline z3/pput.py]: By reading the code it looks to me the env variables have precedence even over `~/.z3.cfg`, which IMHO they should.

**Comment by pdobrescu** [comment]
@AMecea @rciorba can we please get this PR either merged or closed?

**Pull request closed without merging**